fala/apps/adviser/tests/test_results_page_journeys_playwright.py

```diff
@@ -2,20 +2,16 @@
 from fala.playwright.setup import PlaywrightTestSetup
 
 
-class TestEndToEndJourneys(PlaywrightTestSetup):
-
-    def setUp(self) -> None:
-        self.page = self.browser.new_page()
-
-    def visit_valid_uk_results_page(self):
-        self.page.goto(f"{self.live_server_url}")
-        self.page.get_by_label("Postcode").fill("SE11")
-        self.page.get_by_role("button", name="Search").click()
-        expect(self.page.locator("h1")).to_have_text("Search results")
-
+class ResultPageEndToEndJourneys(PlaywrightTestSetup):
```

Comment:
I wonder if it might be worth adding some tests here that check the UI displays as expected for the list of providers? Perhaps checking for the website links and google map links for one provider or something like that.

Reply:
So I think we said that these static views were covered in other tests which included `test_search_view_function.py`. I have changed it's name to `test_fala_views_and_functions.py`